MapboxCoreNavigation/CoreFeedbackEvent.swift

```diff
@@ -26,6 +26,19 @@ class CoreFeedbackEvent: Hashable {
 class FeedbackEvent: CoreFeedbackEvent {
     func update(type: FeedbackType, source: FeedbackSource, description: String?) {
         eventDictionary["feedbackType"] = type.description
+
+        // if there is a subtype for this event then append the subtype description to our list for this type of feedback
+        if let subtypeDescription = type.subtypeDescription {
+            var subtypeList = [String]()
+            if let existingSubtypeList = eventDictionary["feedbackSubType"] as? [String] {
+                subtypeList.append(contentsOf: existingSubtypeList)
+            }
```

Comment:
Is it necessary to merge the existing and new subtypes? FeedbackSubtypeViewController only calls into this code path once all the subtypes have been selected, but technically it would be possible for an application to call `NavigationEventsManager.updateFeedback(uuid:type:source:description:)` multiple times for a single event, which would result in some duplication.

Reply:
see below...